metaserver/src/main/scala/scala/meta/languageserver/providers/DocumentHighlightProvider.scala

```diff
@@ -17,10 +17,10 @@ object DocumentHighlightProvider extends LazyLogging {
   ): DocumentHighlightResult = {
     logger.info(s"Document highlight in ${path}")
     val locations = for {
-      name <- symbolIndex
-        .resolveName(path, position.line, position.character)
+      symbol <- symbolIndex
+        .findSymbol(path, position.line, position.character)
         .toList
-      data <- symbolIndex.symbolIndexer.get(name.symbol).toList
+      data <- symbolIndex.definitionData(symbol).toList
```

Comment:
I think this should be `.referencesData`, because they have different fallback options

Reply:
Done.